src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -1027,8 +1027,8 @@ protected function shouldBlockPhpUpload($value, $parameters)
         }
 
         return ($value instanceof UploadedFile)
-           ? trim(strtolower($value->getClientOriginalExtension())) === 'php'
-           : trim(strtolower($value->getExtension())) === 'php';
+           ? strtolower(trim($value->getClientOriginalExtension())) === 'php'
```

Comment:
why?

Reply:
strtolower on space characters is not making any sense, hence first make input shorter and then normalize it.

Reply:
are there performance implications?  otherwise I see no reason to make this change.